blueprints/ember-cli-moment-shim/index.js

```diff
@@ -1,18 +0,0 @@
-'use strict';
-
-module.exports = {
-  normalizeEntityName: function() {
-    // this prevents an error when the entityName is
-    // not specified (since that doesn't actually matter
-    // to us
-  },
-
-  afterInstall: function() {
-    // we need to install moment-timezone via bower since npmignore
-    // ignores `moment-timezone/builds/*`
-    return this.addBowerPackagesToProject([
```

Comment:
I think we should use `addPackageToProject` to allow the consuming app to govern what version of moment, moment-timezone are pulled in